packages/core/package.json

```diff
@@ -2,7 +2,12 @@
   "name": "@asciidoctor/core",
   "version": "3.0.0-alpha.4",
   "description": "Asciidoctor - the core library",
-  "main": "dist/node/asciidoctor.js",
+  "exports": {
+    ".": {
+     "require": "./dist/node/asciidoctor.cjs",
+     "import": "./dist/node/asciidoctor.js"
+    }
+  },
```

Comment:
Can we keep `main` in addition to `exports`? I believe that some web bundlers/CDNs are using `main` as an hint to tell if the module is ESM-compatible or not.

Reply:
👍🏻 Done